natgeo/generator-ngs
# Update packages and add testing framework

package update:
https://jira.nationalgeographic.com/browse/SYTO-62

testing framework:
https://jira.nationalgeographic.com/browse/SYTO-63

after cloning the repo and ```npm install``` link this local repo as the one Yeoman uses with ```npm link```

To test packages:
* You'll need to setup a test project.
```shell
mkdir [your-test-proj]
cd [your-test-proj]
yo ngs
```
* then run ```npm update``` and shouldn't be any updates

To test framework:
* from your previously created test project, run ```grunt karma``` 
* basic test should run and pass
